.travis.yml

```diff
@@ -10,11 +13,33 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-    
-script:
-  - wget https://raw.githubusercontent.com/sormuras/bach/76983e2f99497738b6f3e9f86bd42fd3a759a275/install-jdk.sh -O $TRAVIS_BUILD_DIR/install-jdk.sh
-  - bash $TRAVIS_BUILD_DIR/install-jdk.sh --feature 11 -L GPL --target $HOME/openjdk11
-  - ./gradlew build codeCoverageReport -PsystemTestJavaHomes="$JAVA_HOME:$HOME/openjdk11"
-after_success:
-  # upload coverage report
-  - bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+
+install:
+  - ./gradlew build :inspectit-oce-agent:systemTest
+
+jobs:
+  include:
+    - stage: deploy
+      name: "Code Coverage Reporting"
+      jdk: oraclejdk8
+      install: ./gradlew build codeCoverageReport
+      after_success:
+        # upload coverage report
+        - bash <(curl -s https://codecov.io/bash)
+    - name: "Documentation Deployment"
+      jdk: oraclejdk8
+      install: ./gradlew :inspectit-oce-documentation:asciidoctor
+      deploy:
+        provider: s3
```

Comment:
Would it be possible to upload the documentation with e.g. a build number in its path, so that the documentation for every PR is accessible? We would then of course most likely need some expiration so that we don't pollute the S3.

What we definetely need is two paths: one for the master and one for the latest release.

Reply:
Let's don't do this yet. Sure you have to build the docs by yourself but otherwise we do have to do all this clean-up in the end.